packages/broker/src/plugins/operator/OperatorPlugin.ts

```diff
@@ -42,6 +43,10 @@ export interface OperatorPluginConfig {
     inspectRandomNode: {
         intervalInMs: number
     }
+    closeExpiredFlags: {
+        intervalInMs: number
+        maxFlagAgeSec: number
```

Comment:
(nitpick) I would keep naming consistent with above
```suggestion
        maxFlagAgeInSec: number
```